src/test/groovy/edu/ie3/datamodel/io/source/csv/CsvRawGridSourceTest.groovy

```diff
@@ -28,8 +29,8 @@ class CsvRawGridSourceTest extends Specification implements CsvTestDataMeta {
   RawGridSource source
 
   def setupSpec() {
-    TypeSource typeSource = new TypeSource(new CsvDataSource(csvSep, typeFolderPath, fileNamingStrategy))
-    source = new RawGridSource(typeSource, new CsvDataSource(csvSep, gridDefaultFolderPath, fileNamingStrategy))
+    TypeSource typeSource = new TypeSource(new CsvDataSource(csvSep, typeFolderPath as Path, fileNamingStrategy))
+    source = new RawGridSource(typeSource, new CsvDataSource(csvSep, gridDefaultFolderPath as Path, fileNamingStrategy))
```

Comment:
Just curious, why is this necessary? The fields should already be of the right type